### DISCUSSION TITLE
.... has permissions 0755, but should have 2775

### DISCUSSION THREAD

**Discussion by robs-fotoalbum** [LycheeOrg/Lychee#1818] [Q&A]
Hello,

another question:
I still have this group permission issue "Warning: .... has permissions 0755, but should have 2775" and "Warning: 5083 more directories with wrong permissions" - I have no idea how to address this as the CHMOD allows not to edit the first position (set 0 to 2)

Cheers
Rob

**Comment by ildyria** [comment]
It seems to me that `chmod 2775 ...` should be working though. No ?

**Reply by robs-fotoalbum** [reply]
Yes, but I cannot set it. How do I do this my FTP doesn't allow changing the first position, it's locked. So I was able to change it from 0755 to 0775, but not to 2775. It's a warning though, I don't know how critical this might be.
